cylc/flow/scripts/workflow_state.py

```diff
@@ -427,6 +429,15 @@ def main(parser: COP, options: 'Values', *ids: str) -> None:
             msg += id_
         else:
             msg += id_.replace(options.depr_point, "$CYLC_TASK_CYCLE_POINT")
+
+        if (
+            options.depr_status
+            and options.depr_status in TASK_STATUSES_ALL
+            and options.depr_status not in TASK_STATUSES_FINAL
+        ):
+            # polling for non-final task stauses is flaky
+            msg += 'and the --triggers option'
```

Comment:
```suggestion
            msg += ' and the --triggers option'
```

Reply:
Although arguably this is unnecessary as it is adequately covered by the `InputError`